c/parallel/src/nvrtc/command_list.h

```diff
@@ -156,9 +156,23 @@ struct nvrtc_command_list_visitor
 
     check(jitlink_error);
 
-    check(nvJitLinkGetLinkedCubinSize(jitlink.handle, &cleanup.cubin_ref.size));
+    bool ptx    = false;
+    auto result = nvJitLinkGetLinkedCubinSize(jitlink.handle, &cleanup.cubin_ref.size);
```

Comment:
Maybe rename `cubin_ref`? E.g. `linked_ref`?

Reply:
Renamed in https://github.com/NVIDIA/cccl/pull/3462/commits/342e894f51db43c3b70cf8dc82e7fc341928e508.